core/dbt/main.py

```diff
@@ -1084,6 +1084,17 @@ def parse_args(args, cls=DBTArgumentParser):
         '''
     )
 
+    p.add_argument(
+        '-q',
+        '--quiet',
+        action='store_true',
+        default=None,
+        help='''
+        Supress all non-error logging during dbt execution. Output from
+        {{ print() }} macro are still displayed.
```

Comment:
I just wanted to be clear about the stdout part, fixed a spelling mistake, and shifted wording around a bit. If anyone has a better way to phrase this I'm down for that too. 
```suggestion
        Suppress all non-error logging to stdout. Does not affect 
        {{ print() }} macro calls.
```

Reply:
Updated!